nexus-api/src/routes/v0/endpoints.rs

```diff
@@ -47,7 +47,11 @@ pub const STREAM_TAGS_REACH_ROUTE: &str =
 // -- SEARCH endpoints --
 const SEARCH_PREFIX: &str = concatcp!(VERSION_ROUTE, "/search");
 pub const SEARCH_USERS_ROUTE: &str = concatcp!(SEARCH_PREFIX, "/users");
-pub const SEARCH_TAGS_ROUTE: &str = concatcp!(SEARCH_PREFIX, "/tags/{label}");
+pub const SEARCH_POSTS_BY_TAG_ROUTE_BASE: &str = concatcp!(SEARCH_PREFIX, "/posts/by_tag");
+pub const SEARCH_POSTS_BY_TAG_ROUTE: &str = concatcp!(SEARCH_POSTS_BY_TAG_ROUTE_BASE, "/{tag}");
+pub const SEARCH_TAGS_BY_PREFIX_ROUTE_BASE: &str = concatcp!(SEARCH_PREFIX, "/tags/by_prefix");
+pub const SEARCH_TAGS_BY_PREFIX_ROUTE: &str =
+    concatcp!(SEARCH_TAGS_BY_PREFIX_ROUTE_BASE, "/{prefix}");
```

Comment:
If we don't expect more than one `"/posts/by_tag/*"` and more than one `"/tags/by_prefix/*"`, like in this case, we could collapse these into just:
```suggestion
pub const SEARCH_POSTS_BY_TAG_ROUTE: &str = concatcp!(SEARCH_PREFIX, "/posts/by_tag/{tag}");
pub const SEARCH_TAGS_BY_PREFIX_ROUTE: &str = concatcp!(SEARCH_PREFIX, "/tags/by_prefix/{prefix}");
```
This avoid unnecessary verbosity. I see other `const`s like `STREAM_POSTS_ROUTE` and `STREAM_POSTS_BY_IDS_ROUTE` also seem to have this pitfall, would also remove the unnecessary redundancy there as well.

Reply:
Done in 84e97ef844eb9ed091f0d6503be2e663852d1558